### PR TITLE
fix OPENZAAK_DOMAIN with subpath

### LINKED ISSUE #1662: Open Zaak v1.12.3 breaks OPENZAAK_DOMAIN?
### Product versie / Product version

1.12.3

### Omschrijf het probleem / Describe the bug

Taiga Ams 12

With 1.12.1 we had the fix #1592 to add LOOSE_FK_LOCAL_BASE_URLS as a way for Amsterdam to support a secondary domainname using OPENZAAK_DOMAIN via their API gateway without having to perform rewrites.

However with 1.12.3 it seems that OPENZAAK_DOMAIN no longer works (noted today 28-05 in Taiga). I suspect that this is due to the OIDC-related changes in #1588 

### Stappen om te reproduceren / Steps to reproduce

_No response_

### Verwacht gedrag / Expected behavior

_No response_

### PR DESCRIPTION
**Pull request by annashamray**
Fixes #1662

### CODE REVIEW

**Review by alextreme — APPROVED** [review]

File: src/openzaak/tests/test_rewrite_host.py [hunk 1]
```diff
@@ -80,6 +80,19 @@ def test_with_usage_http_forwarded_host(self):
 
         self.assertEqual(response.content, b"kekw")
 
+    @override_settings(
+        OPENZAAK_DOMAIN="api.example.nl",
+        ALLOWED_HOSTS=["api.example.nl", "testserver"],
+        FORCE_SCRIPT_NAME="/ozgv-t",
```
[the remark this answers is not in the document]
Comment (stevenbal) [reply src/openzaak/tests/test_rewrite_host.py:86]: In any case, I tested this locally with docker-compose and setting a `SUBPATh` and this does fix the issue

**Review by stevenbal — APPROVED** [review]

File: src/openzaak/tests/test_rewrite_host.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (annashamray) [reply src/openzaak/tests/test_rewrite_host.py:86]: `SUBPATH` env var is used in `bin/docker_start.sh`as `mount` parameter in `uwsgi` command, which will be translated into wsgi `SCRIPT_NAME` environ variable, so there is no SUBPATH` variable in django. There is `FORCE_SCRIPT_NAME` setting to overrule `SCRIPT_NAME` from wsgi environ, so I used it for testing

[the remark this answers is not in the document]
Comment (stevenbal) [reply src/openzaak/tests/test_rewrite_host.py:86]: Ah I see, thanks for the clarification :)

[the remark this answers is not in the document]
Comment (annashamray) [reply src/openzaak/tests/test_rewrite_host.py:86]: Yeah, `SUBPATH` can be used with docker container, but not with `runserver` django command

**Pull request merged**